liquidity/liquidity.go

```diff
@@ -223,10 +348,48 @@ func (m *Manager) SuggestSwaps(ctx context.Context) (
 		suggestion := rule.suggestSwap(balance, outRestrictions)
 
 		// We can have nil suggestions in the case where no action is
-		// required, so only add non-nil suggestions.
-		if suggestion != nil {
-			suggestions = append(suggestions, suggestion)
+		// required, so we skip over them.
+		if suggestion == nil {
+			continue
 		}
+
+		// Get a quote for a swap of this amount.
+		// TODO(carla): add support for non-fast swaps.
+		quote, err := m.cfg.LoopOutQuote(
+			ctx, &loop.LoopOutQuoteRequest{
+				Amount:                  suggestion.Amount,
+				SweepConfTarget:         m.params.ConfTarget,
```

Comment:
It seems that `SweepConfTarget` isn't set in `swapClientServer.SuggestSwaps` when mapping to `looprpc.LoopOutRequest`?

Reply:
Added this in the next commit, since it has the rpc changes for the new fields as well .